canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-builder.scss

```diff
@@ -36,6 +36,31 @@ $operator-margin: 50%;
 	}
 }
 
+.properties-field-and-values-table-container,
```

Comment:
Instead of overwriting the styling can't we use flexible tables `light` styling option?

Reply:
Yes, we can use `light` prop in Flexible table. I have updated PR to pass `light` props as `!this.props.controller.getLight()`. Could you please let me know if this is what you are referring to above ?